docs/building/building-environment.md

```diff
@@ -0,0 +1,153 @@
+# CREATING A LOCAL BUILDING ENVIRONMENT
+
+> These instructions are meant to describe how to create a local building and
+> execution environment. If you would like to build tracee container(s)
+> image(s), [read this](./building-containers.md) instead.
+
+## QUICK STEPS (**impatient readers**)
+
+* Build and execute `tracee-ebpf`:
+
+  ```
+  $ make -f builder/Makefile.tracee-make alpine-prepare
+  $ make -f builder/Makefile.tracee-make alpine-shell
+
+  tracee@f64bb4a2f0b1[/tracee]$ make -f Makefile.one clean
+  tracee@f64bb4a2f0b1[/tracee]$ make -f Makefile.one tracee-ebpf
+  tracee@f64bb4a2f0b1[/tracee]$ sudo ./dist/tracee-ebpf \
+  	-o option:parse-arguments \
+  	--trace comm=bash \
+  	--trace follow \
+  	--trace event!='sched*'
+  ```
+
+Now, in your host's bash shell, execute a command. You will see all events (but
+scheduler ones) being printed, in "table format", to stdout.
+
+* Build and execute `tracee`:
+
+  ```
+  $ make -f builder/Makefile.tracee-make alpine-prepare
+  $ make -f builder/Makefile.tracee-make alpine-shell
+
+  tracee@f64bb4a2f0b1[/tracee]$ make -f Makefile.one clean
+  tracee@f64bb4a2f0b1[/tracee]$ make -f Makefile.one all
+  tracee@f64bb4a2f0b1[/tracee]$ sudo ./dist/tracee-ebpf \
+  	-o format:json \
+  	-o option:parse-arguments \
+  	--trace comm=bash \
+  	--trace follow \
+  	--trace event!='sched*' | \
+  	./dist/tracee-rules \
+  	--input-tracee file:stdin \
+  	--input-tracee format:json
+  ```
+
+Now, in your host's bash shell, execute: `strace /bin/ls` and observe tracee
+warning you about a possible risk (with its Anti-Debugging signature).
+
+Now, for **more patient readers** ...
+
+## HOW TO BUILD AND USE THE ENVIRONMENT
+
+In order to have a controlled building environment for tracee, tracee provides
+a [Makefile.tracee-make](../../builder/Makefile.tracee-make) file that allows you
```

Comment:
Relative paths should be used only to refer other *.md files under the `docs` directory.

For source code we should use absolute URLs with git revision placeholder:

```
[Makefile.tracee-make]: https://raw.githubusercontent.com/aquasecurity/tracee/{{ git.tag }}/builder/Makefile.tracee-make
```